helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -212,6 +223,8 @@ public Job(@JsonProperty("id") final JobId id,
     this.securityOpt = Optional.fromNullable(securityOpt).or(EMPTY_SECURITY_OPT);
     this.networkMode = Optional.fromNullable(networkMode).orNull();
     this.metadata = Optional.fromNullable(metadata).or(EMPTY_METADATA);
+    this.capAdds = Optional.fromNullable(capAdds).or(EMPTY_CAPS);
+    this.capDrops = Optional.fromNullable(capDrops).or(EMPTY_CAPS);
```

Comment:
Thanks. I haven't read much of the Guava docs. But are they saying `firstNonNull()` is better than `or()`? It reads like both a fine.

> Whenever you want a null value to be replaced with some default value instead, use Objects.firstNonNull(T, T). As the method name suggests, if both of the inputs are null, it fails fast with a NullPointerException. If you are using an Optional, there are better alternatives -- e.g. first.or(second).